packages/shared/src/components/modals/BasePostModal.tsx

```diff
@@ -57,6 +73,11 @@ function BasePostModal({
           kind={Modal.Kind.FlexibleTop}
           portalClassName={styles.postModal}
           id="post-modal"
+          overlayRef={(node) => {
+            if (node) {
+              node.addEventListener('scrollend', onScroll);
+            }
+          }}
```

Comment:
Don't we need somewhere to remove the event listener too?

Reply:
@capJavert Do you know, in this scenario with inline ref and node assigned listener we don't need manual cleanup correct?
Node will destroy and cleanup listener?

Other way would be using else, but don't know what to cleanup then.